hw/mcu/dialog/da14699/include/mcu/mcu.h

```diff
@@ -119,6 +119,28 @@ typedef enum {
 #define MCU_GPIO_PORT1(pin)		((1 * 32) + (pin))
 #define MCU_DMA_CHAN_MAX                    8
 
+#define MCU_GP_ADC_SEL0               MCU_GPIO_PORT1(9)
```

Comment:
I'd use common prefix like `MCU_PIN` for all definitions, it will be easier to manage this in case we add more pins. also use `GPADC` and `SDADC` since this is how drivers are named. so this one would be `MCU_PIN_GPADC_SEL0`

Reply:
@JustineKH I agree with andrzej about the naming. If you will make those changes and reflect the same changes to mynewt_runtime #69 then I think we should get this merged ASAP.